src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml.cs

```diff
@@ -135,15 +144,106 @@ private void KeyEventHandler(int key, bool matchValue, int matchValueCode, strin
             }
         }
 
+        private bool FilterAccessibleKeyboardEvents(int key, UIntPtr extraInfoIgnoreKeyEvent)
+        {
+            // If the current key press is tab, based on the other keys ignore the key press so as to shift focus out of the hotkey control.
```

Comment:
We'll still have to send the extraInfo either ways, so the function would still have two arguments. Am I missing something?

Reply:
I modified the code so that the check happens only in the FilterAccessibleKeyboardEvents function. However, as stated in the above comment I have to pass the extra info as an argument.

Reply:
Rather than passing key and extraInfo separately we could just directly pass the entire KeyboardEvent object as is. By doing that in the future the `wparam` member can also be accessed if needed.

Reply:
We can't access `KeyboardEvent` in HotketSettingsControl.xaml.cs because it is a part of the interop class and adding a reference to the interop project would lead to a circular reference.